src/Compute/Compute/Common/ComputeClientInstancViewMethod.cs

```diff
@@ -29,6 +29,23 @@ public static class InstanceViewMethod
             return result;
         }
 
+        public static AzureOperationResponse<VirtualMachineExtensionsListResult> ListWithInstanceView(this IVirtualMachineExtensionsOperations iVmExtensionOperations,
+           string rgName, string vmName)
+        {
+            string expand = "instanceView";
+            var result = iVmExtensionOperations.ListWithHttpMessagesAsync(rgName, vmName, expand).GetAwaiter().GetResult();
+
+            foreach (var vmExtension in result.Body.Value)
+            {
+                if (vmExtension.InstanceView == null)
+                {
+                    vmExtension.InstanceView = iVmExtensionOperations.GetWithInstanceView(rgName, vmName, vmExtension.Name).Body.InstanceView;
```

Comment:
Is there a case when List InstanceView does not give an instance view result, but Get InstanceView gives it?

Reply:
From what I test, ListInstanceView never returned the instance view result, but GetInstanceView always returned it.

Reply:
Same behavior on my side, it seemed a bug in the SDK or smth as ListInstanceView never returned the instance view although $expand is right. That's why my not-so-proud-about-it loop but workaround the limitation.

Reply:
This looks like a bug or missing feature of the service.